docs/docs/external/getting-started/test-network/setup-packages.md

```diff
@@ -0,0 +1,270 @@
+
+### Fabric Network
+
+The code for this lies in the `tests/network-setups` folder.
+
+This folder contains code to create and launch networks `network1` and `network2` of identical specifications:
+- Network: 1 peer, 1 peer CA, 1 ordering service node, 1 ordering service CA
+- Single channel named `mychannel`
+- Single contract named `simplestate` (deployed on `mychannel`) that supports simple transactions (`Create`, `Read`, `Update`, `Delete`) involving storage and lookup of <key, value> pairs.
+
+Follow the instructions below to build and launch the networks:
+- Navigate to the `tests/network-setups/fabric/dev` folder.
+- To spin up both network1 and network2 with interoperation chaincode installed, run:
+  ```
+  make start-interop
+  ```
+- (_Note_: If you do not wish to test Fabric-Fabric interoperation, you can choose to install only one of the two networks along with its interoperation chaincode. For `network1`, run `make start-interop-network1`, and for `network2`, run `make start-interop-network2`.)
+
+For more information, refer to the associated [README](https://github.com/hyperledger-labs/weaver-dlt-interoperability/tree/master/tests/network-setups/fabric/dev).
+
+**Troubleshooting Tips**:
+- If you see any errors during the launches, re-check the prerequisites (software installations and credentials). Ensure your network connection is working. As a safe bet, you can retry after cleanup: kill and remove all Docker containers and associated volumes.
+
+### Fabric Client (fabric-cli)
+
+The CLI is used to interact with a Fabric network, configure it and run chaincode transactions to record data on the channel ledger or query data. It is also used to interact with remote networks through the relay in order to trigger an interoperation flow for data request and acceptance.
+
+The `fabric-cli` source code is located in the `samples/fabric/fabric-cli` folder.
+
+#### Prerequisites
+
+If you are using a Linux system, make sure that lib64 is installed.
+
+_Note_: The setup and running instructions below were tested with all Node.js versions from v11.14.0 to v14.17.3.
+
+#### Installation
+
+You can install `fabric-cli` as follows:
+- Navigate to the `samples/fabric/fabric-cli` folder.
+- Create `.npmrc` from template `.npmrc.template`, by replacing `<personal-access-token>` with yours created [above](#package-access-token)..
+- Run the following to install dependencies:
+  ```bash
+  make build
+  ```
+- Use the `fabric-cli` executable in the `bin` folder for [subsequent actions](./ledger-initialization.md).
+
+### Fabric Relay
+
+The relay is a module acting on behalf of a network, enabling interoperation flows with other networks by communicating with their relays.
+The code for this lies in the `core/relay` folder.
+
+#### Running Relay in Host
```

Comment:
Same as above (remove this heading, increase size of subsequent headings)

Reply:
Fixed